Tasks/AzureResourceManagerTemplateDeploymentV3/operations/DeploymentScopeBase.ts

```diff
@@ -68,6 +71,20 @@ export class DeploymentScopeBase {
         }
     }
 
+    private getAzurePortalDeploymentURL() {
+        let portalUrl = this.taskParameters.endpointPortalUrl ? this.taskParameters.endpointPortalUrl : "https://portal.azure.com";
```

Comment:
the logic in this function should be under a try-catch will suppressed exception so that any error in this function does not show up to user.

Reply:
suppression is necessary, otherwise this will mask the real error for task

Reply:
Yep, done